### ISSUE TITLE
Post-proces SURVIVOR output with vcf2bedpe.R 

### ISSUE THREAD

**Issue by arnikz** [GooglingTheCancerGenome/CNN#82]
In [this](https://github.com/GooglingTheCancerGenome/sv-channels/blob/2c0226066edf1d9d5494f1378dce53195737eb2d/doc/sv-channels.svg) workflow, _vcf2bedpe.R_ is only used to convert SV callers' output. According to https://github.com/GooglingTheCancerGenome/sv-gen/issues/43, we also need to post-process _SURVIVOR simSV_ output but this script seems missing in the workflow.

**Comment by lsantuari** [comment]
I will update the workflow (related to #68).

**Comment by arnikz** [comment]
[StructuralVariantAnnotation](https://bioconductor.org/packages/release/bioc/vignettes/StructuralVariantAnnotation/inst/doc/vignettes.html#loading-data-from-vcf) (SVA) package supports the following VCF notations:

- [ ] Non-symbolic allele
- [x] Symbolic allele with SVTYPE of DEL, INS, and DUP.
- [x] Breakpoint notation SVTYPE=BND
- [ ] Single breakend notation

Currently, we use the two notations (BND for TRA/CTX?) in the workflow.

**Comment by arnikz** [comment]
SVA supports non-compliant VCFs:

- Pindel (SVTYPE=RPL)
- Manta (INv3, INV5 fields)
- Delly (SVTYPE=TRA, CHR2, CT fields)
- TIGRA (SVTYPE=CTX)

Is it correct to say that _vcf2bedpe.R_ script (and _bedpe2vcf.py_) is needed to handle also LUMPY and SURVIVOR output?

**Comment by lsantuari** [comment]
> [StructuralVariantAnnotation](https://bioconductor.org/packages/release/bioc/vignettes/StructuralVariantAnnotation/inst/doc/vignettes.html#loading-data-from-vcf) (SVA) package supports the following VCF notations:
> 
> * [ ]  Non-symbolic allele
> * [x]  Symbolic allele with SVTYPE of DEL, INS, and DUP.
> * [x]  Breakpoint notation SVTYPE=BND
> * [ ]  Single breakend notation
> 
> Currently, we use the two notations (BND for TRA/CTX?) in the workflow.

Yes. I miss the inversions (INV) in the SVA documentation. 

Based on my tests SVA supports also INV (see [page 3 of VCF specs](https://samtools.github.io/hts-specs/VCFv4.3.pdf)).
INV can also be represented in BND notation (example to be added here based on [this](https://simpsonlab.github.io/2015/06/15/merging-sv-calls/)).

**Comment by arnikz** [comment]
> Yes. I miss the inversions (INV) in the SVA documentation.

True. Nevertheless, INVs are handled, for example, in _vcf2bedpe.R_, [`label_classes.py`](https://github.com/GooglingTheCancerGenome/sv-channels/blob/dev/scripts/genome_wide/label_classes.py) and/or [notebook](https://github.com/GooglingTheCancerGenome/notebooks/blob/nb-sv-callers-1.1.0/example_2/sv_evaluate.ipynb) [8].

**Comment by arnikz** [comment]
The script is used to process Manta, DELLY, LUMPY, GRIDSS and SURVIVOR VCF files. However, the following lines do not make it clear for the latter (e.g., TRA vs. CTX).

https://github.com/GooglingTheCancerGenome/sv-channels/blob/50b1a55fa7ca1143f7fcbabc5aae129c6f258cf1/scripts/R/vcf2bedpe.R#L14

https://github.com/GooglingTheCancerGenome/sv-channels/blob/50b1a55fa7ca1143f7fcbabc5aae129c6f258cf1/scripts/R/vcf2bedpe.R#L52

https://github.com/GooglingTheCancerGenome/sv-channels/blob/50b1a55fa7ca1143f7fcbabc5aae129c6f258cf1/scripts/R/vcf2bedpe.R#L71

https://github.com/GooglingTheCancerGenome/sv-channels/blob/50b1a55fa7ca1143f7fcbabc5aae129c6f258cf1/scripts/R/vcf2bedpe.R#L95-L96

Moreover, all issues reported [here](https://github.com/GooglingTheCancerGenome/sv-gen/issues/43) should be handled by this script. Correct?
